src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.figure.Circle;
+import core.basesyntax.figure.IsoscelesTrapezoid;
+import core.basesyntax.figure.Rectangle;
+import core.basesyntax.figure.RightTriangle;
+import core.basesyntax.figure.Square;
+import core.basesyntax.template.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        switch (random.nextInt(5)) {

```

Comment:
**All magic numbers in your code should be constants.**
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced